docs/advanced/api_examples.yml

```diff
@@ -253,4 +254,9 @@
   response: |
     HTTP/1.1 200 OK
 
-    [{"dataset_id": "47440eff-e933-4de0-b56c-d3469b61421f", "primary": "%(NODE_0)s", "maximum_size": 1073741824, "metadata": {}, "deleted": false}]
+    [{"dataset_id": "47440eff-e933-4de0-b56c-d3469b61421f",
+      "primary": "%(NODE_0)s",
+      "maximum_size": 1073741824,
+      "metadata": {},
+      "deleted": false,
+      "path": "/flocker/47440eff"}]
```

Comment:
regarding the reformatting of this line, thank you.  in general, it would be nice if the doc renderer pretty-printed our json for us, I bet.  This is an easier fix though.